### ISSUE TITLE
A view option in menu bar.

### ISSUE THREAD

**Issue by shubhankarsharma00** [CircuitVerse/CircuitVerse#1966]
**Is your feature request related to a problem? Please describe.**
Right now all the options are on the canvas and it might feel a bit cluttered to some users.

**Describe the solution you'd like**
View option in the menubar next to tools with 3 check list options to view the Circuit elements, Timing Diagram help and properties dialog box.

**Are you working on this? (Yes/No)**
No.

**Comment by satu0king** [comment]
The preview circuit feature is there. Do check that once.

**Comment by shubhankarsharma00** [comment]
Timing diagram still appears in preview mode.

**Comment by kshitij86** [comment]
@satu0king I would like to work on this feature request and start contributing to `CircuitVerse` ! 
Can I please get assigned?

**Comment by ShreyaPrasad1209** [comment]
@shubhankarsharma00 maybe we can use preview feature to hide timing diagram as well?

**Comment by shubhankarsharma00** [comment]
I think many will want to use preview feature with the timing diagram for presentations. I was just thinking an option to show it or hide it from the tool bar. lie Microsoft word has a view option in which you can decide which tools you want to view.

**Comment by ShreyaPrasad1209** [comment]
Okay so you want a feature which can specifically hide certain tools?

**Comment by shubhankarsharma00** [comment]
![ezgif com-gif-maker](https://user-images.githubusercontent.com/32474302/102178069-c2f3b700-3eca-11eb-8fa1-278222eac9d6.gif)
I was thinking of something like this.

**Comment by kshitij86** [comment]
@shubhankarsharma00 where would this be implemented? Like the toolbar on the right has the tools.

**Comment by ShreyaPrasad1209** [comment]
@shubhankarsharma00 yup got it. I'm working on it. I have created only a button till now, will add functionality. Let me know if Full Screen button's position is fine. I will change its name later on.

![image](https://user-images.githubusercontent.com/43600306/102179339-f8999f80-3ecc-11eb-8449-7cb896b5743d.png)

**Comment by kshitij86** [comment]
@ShreyaPrasad1209 Can I also contribute to this?

**Comment by ShreyaPrasad1209** [comment]
@kshitij86 sure. Do you want to coordinate on Slack?

**Comment by kshitij86** [comment]
@ShreyaPrasad1209 sure.

**Comment by ShreyaPrasad1209** [comment]
I've added good first issue for a special project and is currently not open for contributors.

**Comment by awwmawno** [comment]
I would like to try working on this issue for Open Source Day '21.

**Comment by gr455** [comment]
> I would like to try working on this issue for Open Source Day '21.

Go ahead!

**Comment by Prerna-0202** [comment]
I would like to work on this issue. 
Is anyone working on this issue?

**Comment by SOUMOJIT-CHAKRABORTY** [comment]
Hi! I am new to opensource can anyone suggest me how to start the journey?....please help

**Comment by pulkit-30** [comment]
Hey @gr455 ,
I can solve this Issue, May you please assign me this issue and please help me with some code pointers and references so that I can start working on this.

**Comment by RahulSingh210** [comment]
I would like to work on this issue if it's still not solved. I'm new to open source so please help me out with the references to start working on this issue.

**Comment by pulkit-30** [comment]
@RahulSingh210, My pr is under review for this issue. Please try on a different one...

**Comment by RahulSingh210** [comment]
@pulkit-30 Ok...I'll look for another one.

**Comment by Sayan-Maity** [comment]
> @RahulSingh210, Can i work on this issue or it is still under review

**Comment by pulkit-30** [comment]
It is under review

**Comment by Sayan-Maity** [comment]
okay, thank you for informing me

**Comment by Sayan-Maity** [comment]
Sorry but , it is under review means only PR ios left  or you would
entertain any other contributors to contribute in it.  ??

**Comment by SethiVanshika** [comment]
@ShreyaPrasad1209  could you assign this issue to me?

**Comment by vedant-jain03** [comment]
> @ShreyaPrasad1209 could you assign this issue to me?

PR for the issue is under review.

**Comment by SethiVanshika** [comment]
> > @ShreyaPrasad1209 could you assign this issue to me?
> 
> PR for the issue is under review.

Okay

**Comment by rinkydevi** [comment]
i can contribute in this issue, how can i proceed?

**Comment by pulkit-30** [comment]
Hey @rinkydevi,
I left a comment, let's wait for their suggestions
Thanks :D

**Comment by rinkydevi** [comment]
@pulkit-30 thanks for your response,
will surely wait for it!

**Comment by Kgupta62** [comment]
Hi 👋

I’d like to work on this issue and add the MongoDB version check instructions to the README for Ubuntu users.
I have 2+ years of industry experience in a product-based company, have contributed to multiple open-source projects, and have hands-on experience working with MongoDB.

Please assign this issue to me so I can start working on it and submit a PR.

Thanks!

**Comment by rajatk281** [comment]
Hi maintainers

I’d like to work on this issue if it’s still open.  
I understand the motivation behind this feature — reducing canvas clutter by giving users more control over what UI elements are visible.

I like the idea of introducing a “View” option in the menubar with toggle/check options for elements such as Circuit Elements, Timing Diagram help, and the Properties dialog. This feels consistent with common UI patterns and would improve usability, especially for advanced users.

I’ve also noted the mention of the existing preview circuit feature and will review it to see how it might relate or be reused while implementing this.

If this approach aligns with the maintainers’ expectations, please let me know and I’d be happy to proceed or share a brief implementation plan before starting.

Thanks!
